contracts/DefaultStrategy.sol

```diff
@@ -522,248 +436,288 @@ contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Ma
     }
 
     /**
-     * @notice Returns the active groups that can receive the entire `votes`
-     * amount based on their current receivable votes limit in Election.sol.
-     * @param votes The number of votes that would potentially be added.
-     * @return The list of votable active groups.
+     * @notice Rebalances CELO between groups that have an incorrect CELO-stCELO ratio.
+     * FromGroup is required to have more CELO than it should and ToGroup needs
```

Comment:
For consistency, prefer to refer to variables with their real capitalization, also consider putting in backticks. Here and elsewhere.